crates/holochain/tests/tests/conductor_services/mod.rs

```diff
@@ -29,6 +29,7 @@ async fn initialize_dpki() {
 /// Crude check that an agent without the same DPKI instance as others can't
 /// validate actions
 #[tokio::test(flavor = "multi_thread")]
+#[cfg_attr(target_os = "windows", ignore = "flaky")]
 async fn validate_with_dpki() {
```

Comment:
Not just windows, this regularly fails for me locally while waiting for consistency